datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -576,138 +568,131 @@ impl std::fmt::Debug for RowAggregationState {
     }
 }
 
-/// Create a RecordBatch with all group keys and accumulator' states or values.
-#[allow(clippy::too_many_arguments)]
```

Comment:
likewise here, moved from a free function to a member function on `GroupedHashAggregateStream`